docs/tables/aws_securityhub_hub.md

```diff
@@ -0,0 +1,44 @@
+# Table: aws_securityhub_hub
+
+Security Hub provides a comprehensive view of the security state of AWS environment and resources. It also provides the readiness status of environment based on controls from supported security standards.
```

Comment:
To run Security Hub we have to have nite indicating that AWS Config is configured. Else you get error as 
```
AWS Config is not appropriately enabled on some accounts
AWS Config is required for Security Hub's security checks. Review remediation steps for the related findings for CIS 2.5. If you recently enabled AWS Config, note that it can take up to 12 hours for Security Hub to detect the change.
```
Add a note as below 
Note:  Please make sure that, AWS Config is appropriately enabled for the account.